ThingIFSDK/ThingIFSDK/IoTOperations/IoTRequestOperation.swift

```diff
@@ -13,7 +13,9 @@ internal enum HTTPMethod: String {
     case GET = "GET-deprecated"
     case get = "GET"
     case post = "POST"
-    case PUT = "PUT"
+    @available(iOS, deprecated: 1.0, message: "use put")
+    case PUT = "PUT-deprecated"
```

Comment:
@yongpingchen The version is iOS version. It is not our SDK version. We can use `PUT` in any versions of iOS so we can not clearly specify the version. Changing `PUT` to `put` comes from API guide line.

We must change all only uppercase enum element to camel case. My point for using `@availabe` is pointing out the code to change in XCode.

According to above opinion, version number is not important in my position.

What do you think?